source/getting-started/quick-start.md

```diff
@@ -247,7 +249,7 @@ all.
 
 So far, your application is listing data, but there is no way for the user
 to interact with the information.
-In web applications, you often want to listen for user events like clicks or hovers.
+In web applications you often want to listen for user events like clicks or hovers.
```

Comment:
Obviously the most contentious change. If I inadvertently start a bike shed, I will withdraw the suggestion. ;)